pkg/buildpack/multi_architecture_helper_test.go

```diff
@@ -0,0 +1 @@
+package buildpack_test
```

Comment:
Do we plan to add tests to this file? Or should we remove it?

Reply:
Yes, I should, I didn't do it when I was developing the PoC

Reply:
Just added some tests